### ISSUE TITLE
No Navigation Menu on left after upgrading from 2.2.6 to 3.0.9

### ISSUE THREAD

**Issue by boozus** [formtools/core#412]
Upgraded from 2.2.6 to 3.0.9 and now the left main navigation menu is no longer visible.  Just a blank pane.  The module nav menu will show up if for example I manually specify the /formtools/modules/system_check/ modules in the url, but the main nav menu is always blank and will now show up.  No errors are thrown anywhere for this that I can see.  I've tried this in both Firefox and Chrome, same issue.

**Comment by benkeen** [comment]
Hey @boozus, which theme are you using? 

But regardless, try: doing a hard refresh in your browser to clear the cache (or just clear cache manually) to ensure you're getting the latest & greatest CSS files. Maybe clear your browser cache just in case. 

If that doesn't work, you could try deleting the contents of you `[formtools root]/themes/[theme name]/cache` folder, then refreshing that page. I doubt that'll help but it'll ensure all new pages get re-cached.

**Comment by benkeen** [comment]
Hey @boozus, which theme are you using? 

Try switching themes. Does the problem occur on all the themes?

**Comment by benkeen** [comment]
Now that IS strange... could you try the default theme as well? Themes are hooked up to pull from the default theme if they don't have a particular file. I'm wondering if somehow your default theme has gotten messed up.

**Comment by boozus** [comment]
Ok, switched to default theme and the main nav menu now shows up.  Switching to any of the other themes make it disappear.  I've left it on the default theme for now, but if there are any suggestions on to fix the other themes would be appreciated.

**Comment by benkeen** [comment]
Hmm! On any page where the sidebar should show up (but doesn't) could you view the source on the page (Command-Option-U on Chrome/Mac) [https://www.computerhope.com/issues/ch000746.htm] and see if you can see the items in the sidebar listed there? If you're not sure where to look, just email me the file at ben.keen@gmail.com and I'll take a look.

**Comment by boozus** [comment]
@benkeen, when I switch to default, I can see my menu items in source:

    `<div class="menu_items">            
      <div class="nav_link"><a href="/formtools/admin/forms/" class="no_border">Forms</a></div>             
      <div class="nav_link"><a href="/formtools/admin/clients/" class="no_border">Clients</a></div>             
      <div class="nav_link"><a href="/formtools/admin/account" class="no_border">Your Account</a></div>              
      <div class="nav_link"><a href="/formtools/admin/modules/" class="no_border">Modules</a></div>              
      <div class="nav_link"><a href="/formtools/admin/settings" class="no_border">Settings</a></div>                 
      <div class="nav_link"><a href="/formtools/index.php?logout" class="no_border">Logout</a></div>                
      <div class="nav_link"><a href="/formtools/admin/themes/index.php" class="no_border">Themes</a></div>
  </div>`
 When I switch to any other team, I see an empty div section for menu items:

 `<div class="menu_items">
  
  </div>`

**Comment by benkeen** [comment]
Thanks @boozus. So yeah, it's not inheriting the default menu for your theme. I'll flag this as a bug & see if anyone else encounters it (please post, if you do!)

To fix it, I think if you copy the `menu.tpl` from the `[form tools root]/themes/default/` folder into the other theme folder, that should fix it. 

Let me know!

**Comment by boozus** [comment]
@benkeen, yep, copying menu.tpl from default into the other themes fixed it.  Much appreciated for your help!

**Comment by benkeen** [comment]
No worries! Interesting it doesn't inherit the file on your system - good to know...
